### PR TITLE
Add test for at-rules

### PR DESCRIPTION
**Pull request by isolovev**

### CODE REVIEW

File: test/autoprefixer.test.js [hunk 1]
```diff
@@ -547,6 +547,13 @@ it('works with CSS Modules', () => {
   postcss([autoprefixer()]).process(':export { selectors: _1q6ho_2 }').css
 })
 
+it('ignore prefix in vendor at rules', () => {
+  let input = read('at-rules')
+  let output = read('at-rules.out')
+  let result = postcss([autoprefixer({ cascade: false })]).process(input)
```
Comment (ai) [inline test/autoprefixer.test.js]: `autoprefixer` will use current browsers and could change output in the future

File: test/autoprefixer.test.js [hunk 2]
```diff
@@ -547,6 +547,13 @@ it('works with CSS Modules', () => {
   postcss([autoprefixer()]).process(':export { selectors: _1q6ho_2 }').css
 })
 
+it('ignore prefix in vendor at rules', () => {
+  let input = read('at-rules')
```
Comment (ai) [inline test/autoprefixer.test.js]: We have a shortcut to do this test. You can find it in `hacks` secltion in this test file.

File: test/cases/at-rules.out.css [hunk 3]
```diff
@@ -0,0 +1,33 @@
+body {
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  appearance: none;
```
Comment (ai) [inline test/cases/at-rules.out.css]: I recommend enabling `cascade` here to make test readable and more standard

Comment (isolovev) [reply test/cases/at-rules.out.css]: I fixed that.

File: test/autoprefixer.test.js (hunk 2 above)
Comment (isolovev) [reply test/autoprefixer.test.js]: I fixed that.

File: test/autoprefixer.test.js (hunk 1 above)
Comment (isolovev) [reply test/autoprefixer.test.js]: I fixed that.

**Pull request merged**